configs/edison/config.20161006bugfix.alpha8.A_WCYCL1850S.ne30_oEC_ICG.edison

```diff
@@ -131,6 +131,7 @@ polarPlot = False
 [streamfunctionMOC]
 ## options related to plotting the streamfunction of the meridional overturning 
 ## circulation (MOC)
+maxChunkSize = 1000
```

Comment:
@milenaveneziani, the point of this PR is to allow a user to specify this option on a case by case, or analysis section, basis.  The global value is only used for the case where chunking is set `True` in an analysis file by the developer.

We still have the following in `config.defaults`:

```
 73 # Large datasets can encounter a memory error.  Specification of a maximum
 74 # chunk size `maxChunkSize` can be helpful to prevent the memory error.  The
 75 # current maximum chunk size assumes approximately 64GB of ram and large files
 76 # with a single time slice.
 77 maxChunkSize = 10000
```
under the `[input]` section.

Were you thinking of something different?  I don't think I understand here.